pagopa/io-app
# fix(Firma con IO): [SFEQS-1274] Add padding between signature field title and related checkbox

## Short description
This PR fix the missing padding between signature field title and related checkbox.

![Simulator Screen Shot - iPhone 13 - 2022-12-23 at 16 34 48](https://user-images.githubusercontent.com/49342144/209361123-b85ea220-48a8-499f-8d46-f45632be2073.png)

## List of changes proposed in this pull request
- Update `SignatureField`
- Update snapshot

## How to test
Run a signing flow and check if signature fields title are on multiple lines and padded correctly with related checkbox
